game/src/common/waypoints.rs

```diff
@@ -69,11 +69,25 @@ impl InputWaypoints {
         let mut delete_buttons = Vec::new();
 
         for (idx, waypt) in self.waypoints.iter().enumerate() {
+            let icon = {
+                let text = Text::from(Line(waypt.order.to_string()).fg(Color::WHITE).bold_body());
+                let batch = text.render(ctx);
+                let bounds = batch.get_bounds();
+                let image = Image::from_batch(batch, bounds)
```

Comment:
Haha wow, this is an awesome way to make a circular background for text! That `CornerRounding` is magic.